script.js

```diff
@@ -1,4 +1,52 @@
+var scissors = "scissors";
+var stone = "stone";
+var paper = "paper";
+var totalScore = 0;
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var numberGenerator = Math.random() * 3;
+  var computerOption = Math.floor(numberGenerator);
+  //If input equals 0  computer chooses scissors
```

Comment:
it would be better if you give computerOption an object rather than numbers.
Since you generate random number from 0 to 2.
you could do this 
var computerChoice ="":
if(computerOption ==0){
computerChoice = scissors
}if(computerOption ==1){
computerChoice = paper
} else{
computerChoice =stone
}